internal/controller/snapshot/snapshot_adapter.go

```diff
@@ -179,6 +179,13 @@ func (a *Adapter) rerunIntegrationPipelinerunForScenario(scenario *v1beta2.Integ
 	if err != nil {
 		return a.HandlePipelineCreationError(err, scenario, testStatuses)
 	}
+	if a.snapshot.GetLabels()["pac.test.appstudio.openshift.io/event-type"] == "pull-request" {
```

Comment:
Using `HasLabelWithValue(a.snapshot, gitops.PipelineAsCodeEventTypeLabel, gitops.PipelineAsCodePullRequestType)`, `pull_request` not `pull-request`?

Reply:
Like this solution better, thanks!